sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1576,6 +1576,16 @@ object SQLConf {
         "turning the flag on provides a way for these sources to see these partitionBy columns.")
       .booleanConf
       .createWithDefault(false)
+
+  val OPTIMIZE_STRING_COPY_IN_JSON_TUPLE =
+    buildConf("spark.sql.optimizeStringCopyInJsonTuple.enabled")
```

Comment:
The name is misleading. It's for fixing a correctness bug not a perf improvement right?